doc/memory.rst

```diff
@@ -63,8 +63,8 @@ A simple example:
     >>> print(f(1))
     1
 
-  However, when we call it a third time, with a different argument, the
-  output gets recomputed::
+  However, when the functions is called a third time, with a different
```

Comment:
However, calling the function with different parameter executes it and recomputes the output